pnpm-workspace.yaml

```diff
@@ -9,6 +9,9 @@ onlyBuiltDependencies:
   - node-pty
   - protobufjs
   - unrs-resolver
+ignoredBuiltDependencies:
+  # Imported by jsdom-testing-mocks but not needed by us.
+  - puppeteer
```

Comment:
I think this doesn't work as I still get warnings after doing `pnpm install`. It was suppossed to be fixed by https://github.com/pnpm/pnpm/issues/9338. If it's not just me but other people too I'll make a comment under the issue.

Reply:
That's a good find, thanks.